public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,43 @@
+ // [START initialize_firebase_in_sw]
+ // Give the service worker access to Firebase Messaging.
+ // Note that you can only use Firebase Messaging here, other Firebase libraries
+ // are not available in the service worker.
+ importScripts('https://www.gstatic.com/firebasejs/7.12.0/firebase-app.js');
+ importScripts('https://www.gstatic.com/firebasejs/7.12.0/firebase-messaging.js');
+
+ // Initialize the Firebase app in the service worker by passing in
+ // your app's Firebase config object.
+ // https://firebase.google.com/docs/web/setup#config-object
+ firebase.initializeApp({
+    apiKey: "AIzaSyCrA4mq9v926h3aX9mfkLlzUSRbjFude14",
+    authDomain: "lunie-push-notifications.firebaseapp.com",
+    databaseURL: "https://lunie-push-notifications.firebaseio.com",
+    projectId: "lunie-push-notifications",
+    storageBucket: "lunie-push-notifications.appspot.com",
```

Comment:
what is appspot?

Reply:
This is the whole config for all the Google services. Maybe remove unused once for simplicity?

Reply:
Only relevant left